django/contrib/auth/__init__.py

```diff
@@ -11,6 +12,7 @@
 from django.utils.module_loading import import_string
 from django.views.decorators.debug import sensitive_variables
 
+from ...utils.deprecation import RemovedInDjango60Warning
```

Comment:
Could you please use the full path as you did in the test file?

Reply:
Also: deprecation landing in 5.2 will be removed in 6.1 (not in 6.0), and we need to add docs for this. Specifically, we need to add a release note and an entry in the `docs/internals/deprecation.txt` file.